guides/computing_hugging_face_embeddings_gpu.mdx

```diff
@@ -59,7 +59,7 @@ Next, enable the vector store experimental feature:
 
 ```sh
 curl \
-  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -X PATCH 'https://MEILISEARCH_URL/experimental-features/' \
```

Comment:
Hi @guimachiavelli when you use the notation _URL in the end of the env var I already expect it to contain the `https`/`http` (protocol) in it. Otherwise we would be using `_HOST` or `_DOMAIN`.

https://stackoverflow.com/questions/9260218/parts-of-a-url-host-port-path

Reply:
Good catch bruno! My bad on it, Bruno is right on this one from my point of view